meet-data/rpu/1702/meet.csv

```diff
@@ -0,0 +1,2 @@
+Federation,Date,MeetCountry,MeetState,MeetTown,MeetName
+IPL,2017-03-05,Russia,,Klin,Открытый Кубок города Клин
```

Comment:
`Klin Open Cup` would be the idiomatic name. (The literal translation is "Open Cup of the City Klin".)